packages/botonic-react/src/components/message/message-image.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useContext } from 'react'
+
+import { WEBCHAT } from '../../constants'
+import { WebchatContext } from '../../contexts'
+import { resolveImage } from '../../util/environment'
+import { BotMessageImageContainer } from './styles'
+
+interface MessageImageProps {
+  imagestyle: Record<string, unknown>
+  isSentByUser: boolean
+  isSentByAgent: boolean
```

Comment:
I think it's better to use a variable called `sentBy` that is an enum that can have values 'agent', 'bot' and 'user'?

Reply:
Yes, I will refactor it to pass only sentBy and do the check inside the component.